configure

```diff
@@ -188,11 +188,8 @@ while : ; do
         no_naked_pointers=true;;
     -spacetime|--spacetime)
         with_spacetime=true;  with_profinfo=true; profinfo_width=26;;
-    -spacetime-call-counts|--spacetime-call-counts)
-        with_spacetime=true
-        with_profinfo=true
-        profinfo_width=26
-        with_spacetime_call_counts=true;;
+    -disable-call-counts|--disable-call-counts)
+        enable_call_counts=true;;
```

Comment:
Surely the option should be `-enable-call-counts`.

Reply:
This line should be `enable_call_counts=false`. I still want the default to be that they are enabled, so the option here is still called `disable-call-counts`.

Reply:
Sure. I was blindly assuming that the code was right.